From 7dcaf20c2d6d6e7d351f87e9db954c2638e393bf Mon Sep 17 00:00:00 2001
From: David Sernelius <david.sernelius@flir.se>
Date: Mon, 21 Jan 2019 10:34:47 +0100
Subject: [PATCH] Disable build of busfreq_optee.c

Needed assembler code not availabla for i.MX7, and we don't use op-tee.
---
 arch/arm/mach-imx/Makefile | 12 ++++++++----
 1 file changed, 8 insertions(+), 4 deletions(-)

diff --git a/arch/arm/mach-imx/Makefile b/arch/arm/mach-imx/Makefile
index 97af9d698908..36ed126b7013 100644
--- a/arch/arm/mach-imx/Makefile
+++ b/arch/arm/mach-imx/Makefile
@@ -133,9 +133,13 @@ obj-$(CONFIG_SOC_VF610) += mach-vf610.o
 
 obj-$(CONFIG_SOC_LS1021A) += mach-ls1021a.o
 
-ifneq ($(CONFIG_OPTEE)$(CONFIG_SOC_IMX6)$(CONFIG_SOC_IMX7),)
-# Bus frequency by OPTEE OS
-obj-y += busfreq_optee.o
-endif
+#
+# We're not using op-tee and it this part seems not to be implemented
+# for IMX7
+#
+#ifneq ($(CONFIG_OPTEE)$(CONFIG_SOC_IMX6)$(CONFIG_SOC_IMX7),)
+## Bus frequency by OPTEE OS
+#obj-y += busfreq_optee.o
+#endif
 
 obj-y += devices/
-- 
2.17.1

